cpp/tests/io/parquet_test.cpp

```diff
@@ -4465,6 +4471,138 @@ TEST_F(ParquetWriterTest, CheckColumnOffsetIndexStruct)
   }
 }
 
+TEST_F(ParquetWriterTest, CheckColumnIndexListWithNulls)
+{
+  auto null_at_even_idx =
+    cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 2; });
+  auto null_at_idx_3 = cudf::test::iterators::null_at(3);
```

Comment:
I think we typically write `using cudf::test::iterators::null_at;` and then write `null_at(3)` in the appropriate locations.

Reply:
should I get rid of the declaration here and just use `null_at(3)` inline everywhere? (as with `nulls_at({0, 2})` below?)

Reply:
Yup, that would be fine.